docusaurus/docs/iOS/basics/overview.md

```diff
@@ -14,8 +14,6 @@ Building on top of the the Stream Chat API, the Stream Chat iOS component librar
 - Read state and typing indicators
 - Channel and message lists
 - Push (APN or Firebase)
-- Offline storage
-- OSX
```

Comment:
We should not remove these from the General Overview, only the UIKit Overview 👍

Reply:
@jeroenleenarts Can you please put back these 2 options on the "Overview.md" ? These should still be on the general overview